pkg/accounting/accounting.go

```diff
@@ -216,6 +225,37 @@ func (a *Accounting) getPeerBalance(peer swarm.Address) (*PeerBalance, error) {
 	return peerBalance, nil
 }
 
+func (a *Accounting) getPeersBalances() (map[string]int64, error) {
+	peersBalances := make(map[string]int64)
+
+	a.balancesMu.Lock()
+	for peer, balance := range a.balances {
+		peersBalances[peer] = balance.balance
+	}
+	a.balancesMu.Unlock()
+
+	err := a.store.Iterate(balancesPrefix, func(key, val []byte) (stop bool, err error) {
+		k := string(key)
```

Comment:
would be nice to extract everything from here to `L247` into a helper function in the same manner like in `balanceKey()`, just returning a `swarm.Address, error`